ethcore/res/ethereum/easthub.json

```diff
@@ -26,7 +22,11 @@
     "minGasLimit": "0x1388",
     "networkID": "0x7",
     "chainID": "0x7",
+	"eip150Transition": "0x0",
```

Comment:
inconsistent indentation ;)

Reply:
I'm going to fix all spec configs to use tabs instead of spaces given that's what's stated in `.editorconfig`, and the majority of our spec configs are tabbed.